descriptors/ModuleDescriptor-template.json

```diff
@@ -774,6 +774,40 @@
           "unit": "minute",
           "delay": "2"
         },
+        {
+          "methods": [
+            "POST"
```

Comment:
Interface version

Reply:
It is a `_timer` interface, I'm not sure if we should update its version